lab-lindsay/server.js

```diff
@@ -0,0 +1,58 @@
+
+'use strict';
+
+const bodyParser = require('./lib/body-parser');
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+const cowsay = require('cowsay');
+const PORT = process.env.PORT || 3000;
+
+const server = module.exports = http.createServer(function(req, res) {
+  // console.log(req.url)
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+  if(req.method === 'POST') {
+  //NOTE: below is a cURL command that can be run from the terminal when your server is running
```

Comment:
**TODO FOR LOST POINTS**

Please take out stray comments and console.logs. Comments are great, but get in the habit of putting them in your own branch and submitting a clean branch.